app/views/kaminari/_paginator.html.haml

```diff
@@ -0,0 +1,10 @@
+= paginator.render do
+  %nav{ 'aria-label': 'Results pages' }
+    %ul.pagination.justify-content-center
+      = prev_page_tag
+      - each_page do |page|
+        - if page.left_outer? || page.right_outer? || page.inside_window?
+          = page_tag page
+        - elsif !page.was_truncated?
+          = gap_tag
+      = next_page_tag
```

Comment:
> All config and styling was ripped straight from round-three

Was it? This doesn't reflect @frothedoatmilk and I's "responsive" implementation.

Reply:
Ah, I took it from your original PR, not the current generation. I'll update them.